content/millennium/dstu2/medications/medication-administration.md

```diff
@@ -0,0 +1,135 @@
+---
+title: MedicationAdministration | DSTU 2 API
+---
+
+# MedicationAdministration
+
+* TOC
+{:toc}
+
+## Overview
+
+The Medication Administration resource is a record of the event of a patient consuming or otherwise being administered a medication.  Medications Administrations are assumed to be Given unless documented otherwise.
+<br/><br/>
+The following fields are returned if valued:
+
+* [Medication Administration id](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.identifier){:target="_blank"}
+* [Status](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.status){:target="_blank"}
+* [Patient](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.patient){:target="_blank"}
+* [Practitioner](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.practitioner){:target="_blank"}
+* [Encounter](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.encounter){:target="_blank"}
+* [Prescription](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.prescription){:target="_blank"}
+* [Effective Time](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.effectiveTime_x_){:target="_blank"}
+* [Medication](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.medication_x_){:target="_blank"}
+* Details of medication administered:
+  * [Dosage](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage){:target="_blank"}
+  * [Route](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.route){:target="_blank"}
+  * [Quantity](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.quantity){:target="_blank"}
+  * [Site](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.site_x_){:target="_blank"}
+  * [Rate](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.rate_x_){:target="_blank"}
+
+
+<!-- ## Terminology Bindings
+
+<%= terminology_table(:medication_statement, :dstu2) %> -->
+
+<!-- ### Contained Medication Bindings
+
+<%= terminology_table(:contained_medication, :dstu2) %> -->
+
+## Search
+
+Search for MedicationAdministrations that meet supplied query parameters:
+
+    GET /MedicationAdministration?:parameters
+
+_Implementation Notes_
+
+* [MedicationAdministration.medication] may be a reference to a [contained] Medication when the Medication cannot be represented by a CodeableConcept because it contains a unique combination of ingredients. Medications in the system always exist within the context of a MedicationAdministration and cannot be be referenced independently.
```

Comment:
nit: `cannot be be referenced independently.` should be `cannot be referenced independently.`

Reply:
Fixed: https://github.com/cerner/fhir.cerner.com/pull/169/commits/037712040917c1d556c6c8c76363db6eb41412c2#diff-39322533700e1bb799f2f6d703a9ae74L48